src/pairs.rs

```diff
@@ -266,13 +266,14 @@ struct PairList<'a, 'b, T: Rewrite> {
     span: Span,
 }
 
-fn is_ident(expr: &ast::Expr) -> bool {
+fn is_ident_or_lit(expr: &ast::Expr) -> bool {
     match &expr.kind {
         ast::ExprKind::Path(None, path) if path.segments.len() == 1 => true,
+        ast::ExprKind::Lit(_) => true,
```

Comment:
Let's make the match more explicit for bool literals.

Reply:
type checking would reject any literal that's not a bool literal, but shrug, changed it.

Reply:
rustfmt operates at an earlier stage of compilation, so if it parses and could be in the AST then we need to be mindful of that.